1118 add browse to help

dstopp opened the pull request:
- Fixes #1118
- Were any Django, import pipeline, table_schema, or dictionary files modified? Y/N
  - If YES:
    - Database used: opus3_test_210526
    - All Django tests pass: Y
    - FLAKE8 run on modified code: Y/NA
- Were any JavaScript or CSS files modified? Y
  - If YES:
    - JSHINT run on all affected files: Y
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: Y
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:
 - added a menu item to the main help dropdown 'Interpreting Preview Images'
 - added a submenu that includes CIRS, UVIS and VIMS preview txt
 - click on submenu item will open the appropriate preview image .txt file in a new tab and close the help menu

Known problems:

dstopp committed:
Initial changes, adding browse products to menu

-Add Browse Products to menu
-create submenu w/ CIRS, UVIS & VIMS
-click will open current .txt file in new tab

dstopp committed:
Minor changes to the menu labels

-rename menu label
-move left pointer over so words align

dstopp committed:
Minor changes to fix menu close issues.

rfrenchseti reviewed (changes requested):
It's now possible to cover up the bottom menu item ("Citing OPUS") if the browser is too short. This happens both with the larger and the smaller font size. Try 880x500 and 985x275.

dstopp committed:
Minor fixes

-fix jshint complaint;
-fix the menu so the font size changes earlier to prevent the last line from being covered up

dstopp committed:
include the new help menu item in the responsive behaviour

dstopp committed:
minor change to font size to squeeze in last line
